src/utils/removeUnnecessarySpaces.ts

```diff
@@ -0,0 +1,7 @@
+import { isString } from "lodash";
+
+const removeUnnecessarySpaces = (value?: string): string => {
+  return !isString(value) ? "" : value.replace(/\s+/g, ' ').trim()
+};
+
+export { removeUnnecessarySpaces };
```

Comment:
Not saying this is wrong, but why don't we use default exports?

Reply:
It's our rule to use named exports